timeseries/src/autogluon/timeseries/models/local/abstract_local_model.py

```diff
@@ -71,6 +68,10 @@ def __init__(
             self.n_jobs = n_jobs
         else:
             raise ValueError(f"n_jobs must be a float between 0 and 1 or an integer (received n_jobs = {n_jobs})")
+        # Default values, potentially overridden inside _fit()
+        self.use_fallback_model = True
+        self.max_ts_length = 2500
```

Comment:
Shall we adjust it based on the freq? For example, for min data, 2500 is less than 2 days which will not be able to capture weekly trends/seasonality

Reply:
@tonyhoo Currently, the models that we use (`AutoETS`, `AutoARIMA`, `Theta`, `ETS`, `ARIMA`, `Naive`, `SeasonalNaive`) are anyway unable to capture multiple seasonalities - they only capture the seasonality at the `seasonal_period` that we provide. This is at most 24 * 60 = 1440 for minutely data, but usually [much smaller](https://github.com/autogluon/autogluon/blob/master/timeseries/src/autogluon/timeseries/utils/seasonality.py#L5) (<= 24). 

I think that the example you described would apply to models like [MSTL](https://nixtla.github.io/statsforecast/models.html#mstl-1) that consider multiple seasonalities, and I agree that we would need to increase the `max_ts_length` for such models if we add them.

@gradientsky I've moved the parameter override code from the [`_fit` method](https://github.com/autogluon/autogluon/pull/3252/files#diff-4a906aede4e4fa2d1d793970ec678d975a8522832fa38a54ab6ecd726a5a709dR88-R90) to `__init__`.